owasp-top10-2017-apps/a7/comment-killer/killer/Makefile

```diff
@@ -0,0 +1,43 @@
+.SILENT:
+.DEFAULT_GOAL := help
+
+COLOR_RESET = \033[0m
+COLOR_COMMAND = \033[36m
+COLOR_YELLOW = \033[33m
+COLOR_GREEN = \033[32m
+COLOR_RED = \033[31m
+
+PROJECT := A7 Gossip World
+PORT := 10007
+SLEEPUNTILAPPSTARTS := 45
+
+## Installs a development environment using docker-compose
+install: compose msg 
+
+## Runs project using docker-compose
+compose:
+	docker-compose -f deployments/docker-compose.yml down -v --remove-orphans
+	docker-compose -f deployments/docker-compose.yml up -d --build --force-recreate
+
+## Deploys project and view docker logs
+deploy: compose
+	docker logs app-a7 -f
+
+## Prints initialization message after compose phase
+msg:
+	chmod +x deployments/check-init.sh
```

Comment:
To have this `msg` target work, you will have to have this `check-init.sh` as well. A good tip is to have a folder similar to [this one](https://github.com/globocom/secDevLabs/tree/master/owasp-top10-2017-apps/a7/gossip-world/deployments). Don't forget to edit `check-init.sh` to better fit the Comment-Killer app. 😄